content/zh-cn/docs/tasks/extend-kubernetes/configure-aggregation-layer.md

```diff
@@ -289,7 +289,7 @@ The Kubernetes apiserver will use the files indicated by `--proxy-client-*-file`
 1. The connection must be made using a client certificate that is signed by the CA whose certificate is in `--requestheader-client-ca-file`.
 2. The connection must be made using a client certificate whose CN is one of those listed in `--requestheader-allowed-names`. **Note:** You can set this option to blank as `--requestheader-allowed-names=""`. This will indicate to an extension apiserver that _any_ CN is acceptable.
 -->
-Kubernetes apiserver 将使用由 `--proxy-client-*-file` 指示的文件来验证扩展 apiserver。
+Kubernetes apiserver 将使用由 `--proxy-client-*-file` 指示的文件来向扩展 apiserver认证。
```

Comment:
```suggestion
Kubernetes API 服务器将使用由 `--proxy-client-*-file` 指示的文件来认证扩展的 API 服务器。
```
v+n+v+n of original trans is a relatively good pattern. Now, it's hard to say an improvement in my opinion.